Extensions/servant.js

```diff
@@ -1140,16 +1140,13 @@ XKit.extensions.servant = new Object({
 					}
 				}
 
-				if (m_post !== "") {
-					var to_run = function(post) {
-						eval(parameter_fixed + "\n//# sourceURL=xkit/servant/servant" + (new Date()).getTime() + ".js");
-					};
-
-					to_run(m_post);
-				} else {
-					eval(parameter_fixed + "\n//# sourceURL=xkit/servant/servant" + (new Date()).getTime() + ".js");
+				var post = null;
```

Comment:
you'll need to wrap any unused variables in the appropriate eslint comments for the check to pass
```js
/* eslint-disable no-unused-vars */
var post;
/* eslint-enable no-unused-vars */
```